internal/compress/lz4.go

```diff
@@ -108,10 +116,13 @@ type lz4Reader struct {
 	r   io.Reader
 }
 
+// Read returns the number of bytes for read p (0 <= n <= len(p)).
+// If any errors occurs, it will return an error.
 func (l *lz4Reader) Read(p []byte) (n int, err error) {
 	return l.r.Read(p)
 }
 
+// Close closes src and r.
 func (l *lz4Reader) Close() (err error) {
 	return l.src.Close()
```

Comment:
maybe we forget to close `l.r`?

Reply:
`r` is `io.Reader` not `io.ReadCloser`.
we do not have to care about it.